### ISSUE TITLE
venice thoughts

### ISSUE THREAD

**Issue by ilteroi** [LoneGazebo/Community-Patch-DLL#4854]
what can we do about venice in AI hands without making it OP for human players?

1. it seems they have a problem with money. puppet infrastructure cost is normal, but puppets generate only half the gold. venice has a lot of puppets ... now they also have a lot of traderoutes to compensate, but it seems the AI doesn't prioritize building caravans enough. even if they are in negative gpt.

2. they may have a supply problem. at least i see a lot of completely undefendend, relatively isolated puppets. now that may be bad unit positioning on the AI side (and a unique venetian problem because of their fragmented empire), or they simply don't have enough units to go around.

**Comment by bigcat88** [comment]
Second:
Allow to create puppet cities with Venetian Merchant.
That will allow only to build first city near capital(solves unique venetian problem) for AI(and will remove many rage quit for Venetian players due to start problems).

And for the first problem:
Add to Venetian UA: trade routes always return back(without tourism bonus) when opponent declare war.
And add to AI Venetian logic to send more international trade routes(now Venice must not be scared if someone declares war on them).

**Comment by ilteroi** [comment]
well, turning merchants into settlers sounds cheesy ... but i do like the war-resistant trade routes.

also, maybe we could life the "only one trade route per city rule" for routes to venice? that should make them a more attractive trade partner ... and thus safer.

**Comment by ilteroi** [comment]
question: the automatic plundering seems to be a "self-plundering" to beat the enemy to the punch. is that by design?

but: it seems to be skipped by default - everybody recalls their trade units on war declaration, so no potential to give a boost to venice.

**Comment by LoneGazebo** [comment]
@ilteroi yeah so they're auto-recalled, but the first DOW has a surprise round to maybe get some TR pillaging in.

**Comment by ilteroi** [comment]
ok venice is still a loser but i found the place where trade influences the attitude between empires. that definitely needs a balance pass.

**Comment by ilteroi** [comment]
also, venice itself was captured in my test game but no new capital was created for them. is that by design?

**Comment by LoneGazebo** [comment]
@ilteroi sounds like a 🐞

**Comment by ilteroi** [comment]
maybe that was only a graphical problem in observer mode ...

anyway, venice is still failing. how about we give them a second settler at start? call the second city malamocco.

**Comment by TechpriestEnginseer** [comment]
Why don't we just give them two settlers named Eraclea and Malamocco. That way Venice can just build 3 guilds in those 3 cities without having to worry about puppet guilds.

**Comment by LoneGazebo** [comment]
Could do a free settler at two techs, instead of a free MoV.

**Comment by Adonis-VII** [comment]
How would free settler, e.g. Pyramids, work? These could become huge spikes if they gave actual Settlers.

**Comment by pineappledan** [comment]
> That way Venice can just build 3 guilds in those 3 cities without having to worry about puppet guilds.

This is funny because this is exactly the problem I said would happen if we turned off GP generation in puppet cities.

**Comment by ilteroi** [comment]
venice can now use settlers if they get them. meaning they cannot build settlers but they can found cities if they have one. so now it's just a matter of granting them one or two extra settlers somehow.

**Comment by ilteroi** [comment]
some more observations, a comparison with america, who also have 7 cities in my current test game and are similar in score.

1. military
* supply cap venice 23 / used 14
* supply cap america 45 / used 40
* venice has way too few units in comparison. maybe they lost some recently, but they have no money to buy new units, which brings me to the next point

2. economy
* building maintenance venice 488 / income from cities (yield+connections) 330 + 57 --> delta minus 101
* building maintenance america 807 / income from cities (yield+connections) 787 + 75 --> delta plus 55
* even considering trade (venice has 16 trade routes and earns an extra 193 gpt. america has 10 routes and gets 120) venice still is about 80 gpt behind.

considering venice should be swimming in gold, this is not a good picture:

![image](https://user-images.githubusercontent.com/1358966/48972582-c3902b00-f02d-11e8-998f-6189d742655a.png)

3. trade routes yields are crazy, gold is totally secondary ... is this intended?

![image](https://user-images.githubusercontent.com/1358966/48972590-f5a18d00-f02d-11e8-9d78-62b8bd766c5d.png)

4. happiness seems to be a bit too easy. everybody gets fat bonuses all the time.

![image](https://user-images.githubusercontent.com/1358966/48972608-41eccd00-f02e-11e8-80b2-5651ff1086fb.png)

**Comment by TechpriestEnginseer** [comment]
dont say happiness is too easy or you'll get people rifled up again.

**Comment by pineappledan** [comment]
I still maintain that GPP generation should be enabled in Venetian puppets. None of this business with free settlers; that's like throwing more wrenches into a machine to knock out the first wrench.

Trade route catchup culture/science is indeed very high. I had a game where I was 4 techs behind the leader, and was generating 1/3 of my total empire's science off of trade routes to him. I think it's equally true that Gold from trade routes is also depressed in this latest version. TRs, especially to city states, don't hit double digit gold until very late.

**Comment by MoiMagnus** [comment]
Continuing this on the forum, if needed: https://forums.civfanatics.com/threads/venice.540351/page-39
